content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md

```diff
@@ -0,0 +1,370 @@
+---
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard:
+  - projects/kotlin/project-5/live-data-observers
+  soft: []
+ready: true
+submission_type: repo
+title: Data binding with ViewModel and LiveData.
+---
+
+### App overview
+
+In the this project, you develop the GuessTheWord app, beginning with starter code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
```

Comment:
This isn't DRY. I see this text copy-pasted into another file. Rather describe the work only once. The recruits can go read the stuff.

Code should be DRY, and so should documentation

Reply:
I want to stay away from making projects fully dependent on the previous project. I agree with what you are saying, but in this case, as before I don't have time to review projects before they begin the next one. And it is also good that way with the Mobile Syllabuses as they are like tutorials and "spoon-feeding", I fell it's better to have an instruction that is completely new to each project and a new repo so that they don't have an excuse to not start a new project. That is unfortunately the type of people I have to deal with.